src/ec/packages/ec-component-menu-legacy/docs/menu-legacy.md

```diff
@@ -0,0 +1,46 @@
+# ECL Twig - EC Navidation Menu
+
+npm package: `@ecl-twig/ec-component-navigation-menu`
+
+```shell
+npm install --save @ecl-twig/ec-component-navigation-menu
+```
+
+## Inpage navigation
+
+### Parameters
+
+Parameters:
+
+- "title" (string) (default: '')
```

Comment:
parameters do not seems to be up to date (here and in example below)

Reply:
done :-)